### ISSUE TITLE
Map template East Outlet 4 (Bottom Right)

### ISSUE THREAD

**Issue by TachiGaroth** [eugeneration/HappyIslandDesigner#52]
Not sure if this would be the right place to put this but I made a map: the one on the bottom right corner of [this Image](https://i.imgur.com/ltSsS3Q.jpg)

Console output: 
{"version":1,"objects":{"amenities_townhallSprite":[75,56],"amenities_airportBlue":[74,85]},"drawing":{"sand":[[25,16,24,17,24,19,25,20,25,28,27,30,28,30,29,31,40,31,41,32,41,36,40,37,39,37,37,39,37,40,36,41,36,48,37,49,37,51,38,52,38,57,39,58,40,58,41,59,41,60,40,61,40,66,43,69,43,73,38,78,38,85,37,86,37,87,36,88,36,90,35,91,35,92,34,93,30,93,29,94,26,94,25,93,23,93,22,92,18,92,17,93,14,93,13,94,10,94,6,90,6,88,4,86,7,86,7,84,6,85,5,84,4,84,4,83,5,82,5,79,6,78,6,77,7,76,7,69,6,68,6,66,5,65,5,62,4,61,4,59,3,59,2,58,2,56,3,55,4,55,4,53,3,52,3,51,4,50,5,50,5,47,7,45,7,44,9,42,9,40,8,39,8,36,5,33,5,30,6,29,6,26,7,25,7,20,5,18,5,9,6,8,9,8,10,9,10,10,11,11,12,10,15,10,16,11,16,13,18,13,19,12,19,10,20,9,24,9,25,10,26,10,27,11,29,11,31,13,32,13,35,10,35,8,36,7,42,7,43,8,43,9,45,9,48,12,48,13,50,13,51,12,51,10,52,9,56,9,57,10,58,10,59,11,61,11,63,13,64,13,65,12,66,12,67,11,70,11,71,12,73,12,74,11,76,11,77,10,78,10,79,11,79,12,80,13,81,12,82,12,83,11,85,11,86,12,87,12,88,13,90,11,92,11,93,10,94,10,95,11,95,12,96,12,96,10,97,9,98,9,99,8,99,7,100,8,102,8,103,9,103,8,104,7,105,7,107,9,107,10,106,11,108,13,107,14,107,15,106,15,107,16,107,17,108,18,108,19,107,20,108,20,110,22,110,26,109,27,109,29,107,31,107,36,106,37,103,37,101,39,100,38,92,38,89,35,88,35,87,36,83,36,82,35,80,35,77,38,76,38,74,40,73,40,68,45,63,45,62,46,60,46,59,45,59,44,55,40,53,40,50,37,46,37,45,36,45,30,43,28,42,28,41,27,30,27,29,26,29,21,31,19,31,17,30,16],[109,68,111,70,111,73,110,74,108,74,107,73,107,74,105,76,107,78,107,83,106,84,106,85,105,86,104,86,104,87,102,87,100,89,100,91,99,92,98,92,97,93,95,93,90,88,86,88,84,86,84,92,83,93,77,93,76,92,74,92,73,91,73,86,74,85,73,85,65,93,62,93,61,94,59,94,58,95,55,95,54,94,52,94,51,93,51,92,43,92,43,89,42,88,42,80,44,78,45,78,47,76,47,68,46,67,46,66,44,64,44,62,45,61,45,58,42,55,42,50,41,49,41,48,40,47,40,42,41,41,44,41,45,42,49,42,51,44,54,44,55,45,55,46,59,50,64,50,65,49,70,49,78,41,82,41,83,40,89,40,91,42,105,42,107,44,107,49,106,50,106,52,107,53,108,53,110,55,110,56,109,57,109,58,107,60,106,60,106,62,107,63,107,65,108,66,108,68],[68,20,67,19,67,18,66,17,65,17,64,16,63,16,62,17,62,18,63,19,63,20,65,22,67,22,68,21],[84,15,82,17,82,18,83,19,83,22,84,23,86,23,87,22,87,21,91,21,92,22,94,22,95,21,95,17,94,16,92,16,91,17,87,17,87,16,86,15],[52,28,52,29,53,30,53,31,52,32,52,33,53,34,55,34,56,33,58,33,59,32,59,30,58,29,57,29,56,28,56,27,55,26,54,26]],"level1":[[12,34,13,33,13,23,12,22,12,18,13,17,13,14,14,13,96,13,97,14,98,14,99,15,100,15,99,16,99,17,102,20,102,23,101,24,101,26,100,27,100,28,99,29,99,34,100,35,100,38,92,38,89,35,88,35,87,36,83,36,82,35,80,35,77,38,76,38,74,40,73,40,68,45,63,45,62,46,60,46,59,45,59,44,55,40,53,40,50,37,46,37,45,36,45,30,43,28,42,28,41,27,30,27,29,26,29,21,31,19,31,17,30,16,25,16,24,17,24,19,25,20,25,28,27,30,28,30,29,31,40,31,41,32,41,36,40,37,39,37,37,39,37,40,36,41,36,48,37,49,37,51,38,52,38,57,39,58,40,58,41,59,41,60,40,61,40,66,43,69,43,73,38,78,38,84,36,84,35,83,30,83,29,84,27,84,26,83,24,83,23,84,19,84,18,83,15,83,15,82,14,82,13,81,13,79,12,78,12,76,13,75,13,72,12,71,12,70,11,69,11,67,13,65,13,63,11,61,11,60,10,59,10,56,12,54,12,51,13,50,13,46,12,45,12,41,13,41,13,36,12,35],[101,76,99,78,99,81,97,83,63,83,62,84,61,84,60,85,57,85,56,86,53,86,52,85,51,85,49,83,45,83,44,84,42,84,42,80,44,78,45,78,47,76,47,68,46,67,46,66,44,64,44,62,45,61,45,58,42,55,42,50,41,49,41,48,40,47,40,42,41,41,44,41,45,42,49,42,51,44,54,44,55,45,55,46,59,50,64,50,65,49,70,49,78,41,82,41,83,40,89,40,91,42,100,42,100,44,99,45,99,49,100,50,100,52,99,53,99,56,101,58,101,60,99,62,99,66,101,68,109,68,111,70,111,73,110,74,108,74,107,73,106,73,105,72,102,72,101,73],[68,20,67,19,67,18,66,17,65,17,64,16,63,16,62,17,62,18,63,19,63,20,65,22,67,22,68,21],[84,15,82,17,82,18,83,19,83,22,84,23,86,23,87,22,87,21,91,21,92,22,94,22,95,21,95,17,94,16,92,16,91,17,87,17,87,16,86,15],[52,28,52,29,53,30,53,31,52,32,52,33,53,34,55,34,56,33,58,33,59,32,59,30,58,29,57,29,56,28,56,27,55,26,54,26]],"level2":[[41,27,30,27,29,26,29,21,31,19,31,17,30,16,25,16,24,17,24,19,25,20,25,28,27,30,28,30,29,31,40,31,41,32,41,34,34,34,32,36,30,36,28,38,28,39,29,40,29,45,30,46,30,50,31,51,31,55,32,56,32,58,28,62,25,62,24,63,19,63,18,62,15,62,14,61,14,60,13,59,13,53,14,52,14,47,13,46,13,37,15,35,15,30,14,29,14,24,13,23,13,16,16,13,49,13,51,15,51,21,49,23,49,32,47,34,45,34,45,30,43,28,42,28],[64,30,63,29,63,28,59,24,59,20,58,19,58,15,60,13,96,13,98,15,98,16,99,17,99,25,98,25,98,30,97,31,90,31,88,29,88,28,87,27,85,27,84,28,83,28,81,30,77,30,76,29,73,29,72,30],[68,20,67,19,67,18,66,17,65,17,64,16,63,16,62,17,62,18,63,19,63,20,65,22,67,22,68,21],[84,15,82,17,82,18,83,19,83,22,84,23,86,23,87,22,87,21,91,21,92,22,94,22,95,21,95,17,94,16,92,16,91,17,87,17,87,16,86,15]],"level3":[[14,17,17,14,36,14,37,15,38,15,39,16,39,21,38,22,38,23,36,25,34,25,33,24,32,24,31,25,29,25,29,21,31,19,31,17,30,16,25,16,24,17,24,19,25,20,25,25,23,25,22,24,21,24,20,23,17,23,16,22,15,22,14,21],[15,40,16,40,17,39,20,39,22,41,23,41,24,42,24,49,25,50,25,55,24,56,24,58,22,60,17,60,15,58,15,49,16,48,16,46,14,44,14,41],[87,17,87,15,88,14,95,14,98,17,98,23,97,24,97,28,95,30,91,30,89,28,89,27,88,26,88,25,87,24,87,21,91,21,92,22,94,22,95,21,95,17,94,16,92,16,91,17]],"rock":[[18,13,19,12,19,10,20,9,24,9,25,10,26,10,27,11,29,11,31,13],[36,9,37,10,37,11,42,11,42,9,45,9,48,12,48,13,32,13],[50,13,51,12,51,10,52,9,56,9,57,10,58,10,59,11,61,11,63,13],[67,11,70,11,71,12,73,12,74,11,76,11,77,10,78,10,79,11,79,12,80,13,81,12,82,12,83,11,85,11,86,12,87,12,88,13,90,11,92,11,93,10,94,10,95,11,95,12,96,12,96,10,97,9,98,9,99,8,99,7,100,8,102,8,103,9,103,8,104,7,105,7,107,9,107,10,106,11,108,13,107,14,107,15,99,15,98,14,97,14,96,13,64,13,65,12,66,12],[12,16,10,14,7,14,5,12,5,9,6,8,9,8,10,9,10,10,11,11,12,10,15,10,16,11,16,13,14,13,13,14,13,15],[11,19,12,20,12,24,10,24,10,23,9,23,9,21],[11,37,12,36,13,36,13,40,12,41,11,40],[8,59,6,61,5,61,4,60,4,59,3,59,2,58,2,56,3,55,4,55,4,53,3,52,3,51,4,50,7,50,9,52,9,54,8,55],[102,37,102,38,101,39,100,38,100,37],[44,89,44,91,43,91,43,89],[35,92,34,91,34,90,35,89,36,90,35,91],[36,86,37,87,36,88,35,87],[72,84,72,83,75,83,75,84,77,84,77,83,85,83,85,86,84,86,84,92,83,93,77,93,76,92,74,92,73,91,73,86,74,85,73,85,73,84],[103,42,105,42,106,43,105,44,104,44,103,43],[102,52,105,52,106,53,108,53,110,55,110,56,109,57,109,58,107,60,106,60,105,59,105,57,103,57,101,55,101,53]],"pathBrick":[1,84,7,84,7,86,1,86],"pathSand":[73,56,83,56,84,57,84,65,83,66,73,66,72,65,72,57],"pathDirt":[]}}
![ACNH Island Template](https://user-images.githubusercontent.com/29523960/78521223-d02ea280-778e-11ea-970a-74f90a00a5ed.png)

**Comment by TachiGaroth** [comment]
Just in case someone wants to make a template:

1. I used a screenshot of the map and put a 16x16 grid over each in-game grid (I had someone do this for me with illustrator)

2. Then opened Happy Island Designer on [Glass Browser](https://www.electronjs.org/apps/glass-browser). This allows you to open a browser with transparency.

3. To match the exact dimensions of both grids I used Win 10 galery to open the screenshot and adjust manually the dimensions with the slider until it matched the size of the grid in the app.

4. Put the glass browser on top of the screenshot, align both grids and trace.

The set up Its a bit cumbersome but it saves a lot of time afterwards and the end result is near perfect.
